pages/accountLists/[accountListId]/contacts.page.tsx

```diff
@@ -1,28 +1,48 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Head from 'next/head';
 import { useTranslation } from 'react-i18next';
 import { useRouter } from 'next/router';
 import { Box } from '@material-ui/core';
 import { ContactFilters } from '../../../src/components/Contacts/ContactFilters/ContactFilters';
 import { ContactsTable } from '../../../src/components/Contacts/ContactsTable/ContactsTable';
+import { ContactDetails } from '../../../src/components/Contacts/ContactDetails/ContactDetails';
 
 const ContactsPage: React.FC = () => {
   const { t } = useTranslation();
   const {
     query: { accountListId },
   } = useRouter();
 
+  const [contactDetailsId, setContactDetailsId] = useState<string | null>(null);
+  const [contactDetailsHidden, setContactDetailsHidden] = useState<boolean>(
+    true,
+  );
+
+  const toggleContactDetailsHidden = (contactId: string) => {
+    setContactDetailsId(contactId);
+    setContactDetailsHidden(!contactDetailsHidden);
+  };
+
   return (
     <>
       <Head>
         <title>MPDX | {t('Contacts')}</title>
       </Head>
-      <Box height="100vh" display="flex">
-        <Box flex={1}>
+      <Box height="100vh" display="flex" overflow="hidden">
```

Comment:
Do we want the filters to scroll instead? Guess there's still work to do here later to make them sticky.
```suggestion
      <Box height="100vh" display="flex" overflow-y="scroll">
```

Reply:
I can try that... It seemed to work fine with `overflow="hidden"`, but I haven't tested it with a lot of screen sizes and scenarios.  We can keep trying things to get the experience that we want.